<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polynomial Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Калькулятор полиномов с комплексными коэффициентами</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="polynom1Input">
      <property name="placeholderText">
       <string>Полином 1 (например: 1+2x+3x^2)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="polynom2Input">
      <property name="placeholderText">
       <string>Полином 2 (для сложения, вычитания, умножения)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Сложение (+)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="subtractButton">
        <property name="text">
         <string>Вычитание (-)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="multiplyButton">
        <property name="text">
         <string>Умножение (×)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="derivativeButton">
        <property name="text">
         <string>Производная (d/dx)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Результат:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="resultDisplay">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>